<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NCM_StarterList</class>
 <widget class="QMainWindow" name="NCM_StarterList">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>458</width>
    <height>801</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_StarterLists"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_StageName">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Stage Name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>458</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data</string>
    </property>
    <addaction name="actionChose_competitors_and_stage"/>
   </widget>
   <widget class="QMenu" name="menuUpdate">
    <property name="title">
     <string>Update</string>
    </property>
    <addaction name="actionUpdate_now"/>
    <addaction name="actionUpdate_automatically"/>
   </widget>
   <widget class="QMenu" name="menuLayout">
    <property name="title">
     <string>Layout</string>
    </property>
    <addaction name="actionChange_table_settings"/>
   </widget>
   <addaction name="menuData"/>
   <addaction name="menuUpdate"/>
   <addaction name="menuLayout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionChose_competitors_and_stage">
   <property name="text">
    <string>Chose competitors and stage</string>
   </property>
  </action>
  <action name="actionUpdate_now">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Update now</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionUpdate_automatically">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Update automatically</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionChange_table_settings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Change table settings</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
